### PR TITLE
Implement the "add elderly" function

### PR DESCRIPTION
**Pull request by heeeyi**
Fixes: #13 
The current version allows users to add an elderly into the database.
The command format:
add_elderly n/Hello Kitty p/82221990 e/test@test.com a/some nr/A1234567S ag/69 r/MEDIUM

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -19,11 +19,11 @@
 import seedu.address.commons.util.CollectionUtil;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
+import seedu.address.model.person.information.Address;
+import seedu.address.model.person.information.Email;
+import seedu.address.model.person.information.Name;
```
Comment (gohyongjing) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:24]: I like how you moved address email and names into their own package to keep it well organised

File: src/main/java/seedu/address/model/person/information/RiskLevel.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            return "low";
+        case MEDIUM:
+            return "medium";
+        case HIGH:
+            return "high";
+        default:
+            return "invalid";
+        }
+
+    }
```
Comment (gohyongjing) [inline src/main/java/seedu/address/model/person/information/RiskLevel.java:52]: Perhaps we can use riskStatus.toString().toLowerCase();
```suggestion
    public String toString() {
        return riskStatus.toString().toLowerCase();
    }
```

**Review by gohyongjing — COMMENTED** [review]
I like how your code follows the style of AB3, which makes it very clean and intuitive to understand
LGTM

**Review by wz2k — COMMENTED** [review]
Excellent work! 👍  Just some minor checkstyle issues like unused imports and wildcard imports. I have only identified the wildcard imports. Do remember to set the checkstyle settings to the local checkstyle file like how you did in the ip. For reference, https://se-education.org/guides/tutorials/checkstyle.html

File: src/main/java/seedu/address/model/person/information/RiskLevel.java (hunk 2 above)
Comment (wz2k) [reply src/main/java/seedu/address/model/person/information/RiskLevel.java:52]: Another option would be to use name()
```
public String toString() {
    return riskStatus.name().toLowerCase();
}
```
The only difference between name() and toString() is that toString() can be overridden.

**Pull request closed without merging**